### PR TITLE
[522] "Tworzenie oferty" link for anonymous user

### PR DESCRIPTION
**Pull request by mlipa**
__Story / Bug id:__ https://github.com/CodeForPoznan/volontulo/issues/522

__Reference person:__ @mlipa

__Description:__

Add redirect to registration page when the user is anonymous.

### CODE REVIEW

File: apps/volontulo/views/offers.py [hunk 1]
```diff
@@ -68,6 +68,16 @@ def get(request):
...
         :param request: WSGIRequest instance
         """
+        if request.user.is_anonymous():
+            messages.info(
+                request,
+                "Aby założyć ofertę musisz się zalogować lub zarejestrować."
+            )
+
+            return redirect(
+                reverse('login') + '?next={path}'.format(path=request.path)
```
Comment (magul) [inline apps/volontulo/views/offers.py]: `'{login}?next={path}'.format(login=reverse('login'), path=request.path)`
?

Comment (mlipa) [inline apps/volontulo/views/offers.py]: Yup would be better this way!

File: apps/volontulo/views/offers.py [hunk 2]
```diff
@@ -68,6 +68,18 @@ def get(request):
 
         :param request: WSGIRequest instance
         """
+        if request.user.is_anonymous():
+            messages.info(
+                request,
+                "Aby założyć ofertę musisz się zalogować lub zarejestrować."
```
Comment (w1stler) [inline apps/volontulo/views/offers.py]: Add comma after "ofertę". Missed that in AC's. ;-)

Comment (mlipa) [inline apps/volontulo/views/offers.py]: Done.

**Pull request merged**